ex03.ex

```diff
@@ -152,10 +164,17 @@ defmodule Ex03 do
 
   Think a little about a nice way to lay this code out.
   """
+  def won({_, x , _, _, x, _, _, x, _}), do: x
```

Comment:
# -1

I was hoping for a 2d layout...

Reply:
That's what I figured. Could we go over this in class by chance tomorrow?